server/src/test/java/com/datastrato/gravitino/server/web/rest/TestMetalakeAdminOperations.java

```diff
@@ -0,0 +1,215 @@
+        UserAlreadyExistsException.class.getSimpleName(), errorResponse1.getType());
+
+    // Test to throw internal RuntimeException
+    doThrow(new RuntimeException("mock error")).when(manager).addMetalakeAdmin(any());
+    Response resp3 =
+        target("/admins")
+            .request(MediaType.APPLICATION_JSON_TYPE)
+            .accept("application/vnd.gravitino.v1+json")
+            .post(Entity.entity(req, MediaType.APPLICATION_JSON_TYPE));
+
+    Assertions.assertEquals(
+        Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), resp3.getStatus());
+
+    ErrorResponse errorResponse2 = resp3.readEntity(ErrorResponse.class);
+    Assertions.assertEquals(ErrorConstants.INTERNAL_ERROR_CODE, errorResponse2.getCode());
+    Assertions.assertEquals(RuntimeException.class.getSimpleName(), errorResponse2.getType());
+  }
+
+  private User buildUser(String user) {
+    return UserEntity.builder()
+        .withId(1L)
+        .withName(user)
+        .withRoles(Collections.emptyList())
+        .withAuditInfo(
+            AuditInfo.builder().withCreator("creator").withCreateTime(Instant.now()).build())
+        .build();
+  }
+
+  @Test
+  public void testRemoveMetalakeAdmin() {
+    when(manager.removeMetalakeAdmin(any())).thenReturn(true);
+
+    Response resp =
+        target("/admins/user1")
+            .request(MediaType.APPLICATION_JSON_TYPE)
+            .accept("application/vnd.gravitino.v1+json")
+            .delete();
+
+    Assertions.assertEquals(Response.Status.OK.getStatusCode(), resp.getStatus());
+    RemoveResponse removeResponse = resp.readEntity(RemoveResponse.class);
+    Assertions.assertEquals(0, removeResponse.getCode());
+    Assertions.assertTrue(removeResponse.removed());
+
+    // Test when failed to remove user
+    when(manager.removeMetalakeAdmin(any())).thenReturn(false);
+    Response resp2 =
+        target("/admins/user1")
+            .request(MediaType.APPLICATION_JSON_TYPE)
+            .accept("application/vnd.gravitino.v1+json")
+            .delete();
+
+    Assertions.assertEquals(Response.Status.OK.getStatusCode(), resp2.getStatus());
+    RemoveResponse removeResponse2 = resp2.readEntity(RemoveResponse.class);
+    Assertions.assertEquals(0, removeResponse2.getCode());
+    Assertions.assertFalse(removeResponse2.removed());
+
+    doThrow(new RuntimeException("mock error")).when(manager).removeMetalakeAdmin(any());
+    Response resp3 =
+        target("/admins/user1")
+            .request(MediaType.APPLICATION_JSON_TYPE)
+            .accept("application/vnd.gravitino.v1+json")
+            .delete();
+
+    Assertions.assertEquals(
+        Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), resp3.getStatus());
+
+    ErrorResponse errorResponse = resp3.readEntity(ErrorResponse.class);
+    Assertions.assertEquals(ErrorConstants.INTERNAL_ERROR_CODE, errorResponse.getCode());
+    Assertions.assertEquals(RuntimeException.class.getSimpleName(), errorResponse.getType());
+  }
+}
```

Comment:
Can you please add tests about authorization set to false.

Reply:
Done.